app/models/event.rb

```diff
@@ -9,4 +9,11 @@ class Event < ApplicationRecord
   def has_member?(user)
     members.include?(user)
   end
+
+  def event_status
+    return '' unless self.started_at
+    return 'in_progress' if self.started_at == Time.now
```

Comment:
It will be valid only for one specific millisecond and normally even take 1h or even more. You need to check if this is after started_at data but no longer then event period.